src/styles/StyleUtils.ts

```diff
@@ -38,11 +40,10 @@ type ButtonSizeValue = ValueOf<typeof CONST.DROPDOWN_BUTTON_SIZE>;
 type EmptyAvatarSizeName = ValueOf<Pick<typeof CONST.AVATAR_SIZE, 'SMALL' | 'MEDIUM' | 'LARGE'>>;
 type ButtonStateName = ValueOf<typeof CONST.BUTTON_STATES>;
 type AvatarSize = {width: number};
-type ParsableStyle = ViewStyle | CSSProperties | ((state: PressableStateCallbackType) => ViewStyle | CSSProperties);
 
 type WorkspaceColorStyle = {backgroundColor: ColorValue; fill: ColorValue};
 
-type ModalPaddingStylesArgs = {
+type ModalPaddingStylesParams = {
```

Comment:
have we settled on the naming convention we use? why `Args` was introduced in the first place?

Reply:
We don't have a convention for this. I think we naturally settled for `Params`